### PR TITLE
Add Integrations::configure

### LINKED ISSUE #580: Add Integrations::configure
We need a configure method in the Integations API. See https://developer.github.com/early-access/integrations/user-identification-authorization/

### PR DESCRIPTION
**Pull request by m1guelpf**
Fix #580

### CODE REVIEW

File: lib/Github/Api/Integrations.php [hunk 1]
```diff
@@ -2,12 +2,32 @@
...
+     * Configure the Accept header depending on the blob type.
+     *
+     * @param string|null $bodyType
+     *
+     * @return self
+     */
+    public function configure($bodyType = null)
+    {
+        if ('raw' === $bodyType) {
+            $this->acceptHeaderValue = sprintf('application/vnd.github.%s.raw', $this->client->getApiVersion());
```
Comment (acrobat) [inline lib/Github/Api/Integrations.php]: I would just set the required header and leave the additional checks away. See: https://github.com/KnpLabs/php-github-api/blob/master/lib/Github/Api/Project/Cards.php#L20

File: lib/Github/Api/Integrations.php [hunk 2]
```diff
@@ -2,12 +2,30 @@
...
 /**
  * @link   https://developer.github.com/v3/integrations/
  * @author Nils Adermann <naderman@naderman.de>
  */
 class Integrations extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    /**
+     * Configure the accept header for Early Access to the projects api
```
Comment (acrobat) [inline lib/Github/Api/Integrations.php]: integrations api instead of projects

File: lib/Github/Api/Integrations.php [hunk 3]
```diff
@@ -2,12 +2,30 @@
...
+     *
+     * @see https://developer.github.com/v3/repos/projects/#projects
```
Comment (acrobat) [inline lib/Github/Api/Integrations.php]: https://developer.github.com/v3/integrations/

**Comment by m1guelpf** [comment]
@acrobat Copy-paste errors... fixed now.

**Pull request merged**